src/php/Lang/Symbol.php

```diff
@@ -8,6 +8,28 @@
 
 final class Symbol extends AbstractType implements IIdentical
 {
+    public const NAME_DEF = 'def';
+    public const NAME_NS = 'ns';
+    public const NAME_FN = 'fn';
+    public const NAME_QUOTE = 'quote';
+    public const NAME_DO = 'do';
+    public const NAME_IF = 'if';
+    public const NAME_APPLY = 'apply';
+    public const NAME_LET = 'let';
+    public const NAME_PHP_NEW = 'php/new';
+    public const NAME_PHP_OBJECT_CALL = 'php/->';
+    public const NAME_PHP_OBJECT_STATIC_CALL = 'php/::';
+    public const NAME_PHP_ARRAY_GET = 'php/aget';
+    public const NAME_PHP_ARRAY_SET = 'php/aset';
+    public const NAME_PHP_ARRAY_PUSH = 'php/apush';
+    public const NAME_PHP_ARRAY_UNSET = 'php/aunset';
```

Comment:
@jenshaase I was thinking between `NAME_PHP_AGET` and `NAME_PHP_ARRAY_GET`. I think the first one is closer to the real symbol itself, but the second one is more expressive, but what do you think?

Reply:
`NAME_PHP_ARRAY_GET` is fine for me